frontend-react/src/pages/submissions/SubmissionsTable.tsx

```diff
@@ -0,0 +1,60 @@
+import { useContext } from "react";
+import moment from "moment";
+import { useResource } from "rest-hooks";
+
+import SubmissionsResource from "../../resources/SubmissionsResource";
+import { GlobalContext } from "../../components/GlobalContextProvider";
+
+function SubmissionsTable() {
+    // this component will refresh when global context changes (e.g. organization changes)
+    const globalState = useContext(GlobalContext);
+    const submissions: SubmissionsResource[] = useResource(
+        SubmissionsResource.list(),
+        { organization: globalState.state.organization }
+    );
+    return (
+        <div className="grid-container usa-section margin-bottom-10">
+            <div className="grid-col-12">
+                <h2>Submissions</h2>
+                <table
+                    className="usa-table usa-table--borderless prime-table"
+                    aria-label="Submission history from the last 30 days"
+                >
+                    <thead>
+                        <tr>
+                            <th scope="col">Date/time submitted</th>
+                            <th scope="col">File</th>
+                            <th scope="col">Records</th>
+                            <th scope="col">Report ID</th>
+                            <th scope="col">Warnings</th>
+                        </tr>
+                    </thead>
+                    <tbody id="tBody" className="font-mono-2xs">
+                        {submissions.map((s, i) => {
+                            if (s == null) {
+                                return <tr key={"submission_" + i} />;
+                            }
```

Comment:
Not sure why, but was running into situations where submissions were coming in null. Failsafe handling here to just add an empty row rather than crash

Reply:
Oh, interesting. so `submissions` was null?

Maybe above the answer is SubmissionsResource.list() should return and empty array if something goes wrong?

What network requests look like?

Reply:
I pulled the branch and ran the code but I'm not seeing any error here. Any other clues as to what might be going wrong from the network requests?

Reply:
Typo above, `submissions` wasn't null, just an individual submission in the array. it was happening for me for A1H data on staging, going to dig into the response a bit more

Reply:
Maybe instead of an empty row you can do a simple "No results" text? I used that on the Daily Data table and it feels pretty clean to me.